docs/conf.py

```diff
@@ -65,3 +65,29 @@
 from sunpy_sphinx_theme.conf import *
 
 master_doc = "index"
+
+# -- Turn on nitpicky mode for sphinx (to warn about references not found) ----
+#
+# nitpicky = True
+# nitpick_ignore = []
+#
+# Some warnings are impossible to suppress, and you can list specific references
+# that should be ignored in a nitpick-exceptions file which should be inside
+# the docs/ directory. The format of the file should be:
+#
+# <type> <class>
+#
+# for example:
+#
+# py:class astropy.io.votable.tree.Element
+# py:class astropy.io.votable.tree.SimpleElement
+# py:class astropy.io.votable.tree.SimpleElementWithContent
+#
+# Uncomment the following lines to enable the exceptions:
+#
+# for line in open('nitpick-exceptions'):
+#     if line.strip() == "" or line.startswith("#"):
+#         continue
+#     dtype, target = line.split(None, 1)
+#     target = target.strip()
+#     nitpick_ignore.append((dtype, six.u(target)))
```

Comment:
Why is this here?

Reply:
It is for in case we will need to use it in the future to turn on `nitpicky`. I have used it for some `astropy` docs before, and thought we might need it someday if our docs grow larger in size and there might be some problems that can only be resolved by `nitpick-exceptions`.